images/base/files/usr/local/bin/entrypoint

```diff
@@ -18,6 +18,23 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+configure_containerd() {
+  # we need to switch to the 'native' snapshotter on zfs
+  if [[ "$(stat -f -c %T /kind)" == 'zfs' ]]; then
+    sed -i 's/snapshotter = "overlayfs"/snapshotter = "native"/' /etc/containerd/config.toml
```

Comment:
this line itself is fine except we forgot to put the explicit snapshotter in the config. https://github.com/kubernetes-sigs/kind/issues/1891